app/helpers/address_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AddressHelper
```

Comment:
Go ahead and delete the helper files that are unused.  They are created when you use generators, you probably wont use them much once you become comfortable with rails.